apps/src/applab/firebaseCounters.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/* global Applab */
+
+const Firebase = require("firebase");
```

Comment:
That does sound right, though I feel like I have seen issues importing ES5 style before too. Again, I think that was before we switched to webpack where things are slightly different.